client/src/pages/Login.js

```diff
@@ -119,7 +141,7 @@ const Login = ({ isLoginForm }) => {
 
   return (
     <div className="text-center">
-      <h1>Welcome</h1>
+      <Title>{isLoginForm ? "Sign In" : "Sign Up"}</Title>
       <form id="login-password" method="POST">
         <InputWrapper>
           <Label style={StyleLabel} label="E-mail" />
```

Comment:
For the individual inputs, the input text needs a bit more styling and space between the line, also looks like the color of the line when it's active is lighter

Current:
<img width="335" alt="Screen Shot 2020-05-03 at 9 27 16 PM" src="https://user-images.githubusercontent.com/42228386/80936184-de1f1580-8d84-11ea-83c7-e83a4d321816.png">


Figma:
<img width="271" alt="Screen Shot 2020-05-03 at 9 25 05 PM" src="https://user-images.githubusercontent.com/42228386/80936160-c182dd80-8d84-11ea-9a0e-e9ce1ae51022.png">
<img width="161" alt="Screen Shot 2020-05-03 at 9 28 05 PM" src="https://user-images.githubusercontent.com/42228386/80936220-fdb63e00-8d84-11ea-8c07-704ecb5477e9.png">